.gitignore

```diff
@@ -1,3 +1,5 @@
 /bazel-*
 **__pycache__**
 .pytype
+Manhattan_data.txt
+Pittsburgh_data.txt
```

Comment:
Please remove these from gitignore, and have a working area outside of your repository for working with data like this.

Reply:
done